### DISCUSSION TITLE
why you used android build sever in setting.gradle?

### DISCUSSION THREAD

**Discussion by cloner93** [android/nowinandroid#378] [General]
hi 
I want figure out why you use android build server in setting.gradle.

`
        maven { url = uri("../nowinandroid-prebuilts/m2repository") }
`

**Comment by khalid64927** [comment]
This is not in settings.gradle it is in project build gradle and its most likely the path of dependencies on build server

**Comment by Jaehwa-Noh** [comment]
It was added in https://github.com/android/nowinandroid/pull/205/commits/a6250811ec8cd09f0a5785c5de5360190b744f32
This find a maven dependency at `../nowinandroid-prebuilts/m2repository` and `mavenCentral()`

**Comment by dturner** [comment]
This path is only used internally as part of our Google Play build and release process. I have updated the comment to make this clearer.
